airbyte-integrations/connectors/destination-s3-data-lake/src/main/kotlin/io/airbyte/integrations/destination/s3_data_lake/TableIdGenerator.kt

```diff
@@ -20,24 +20,33 @@ interface TableIdGenerator {
     fun toTableIdentifier(stream: DestinationStream.Descriptor): TableIdentifier
 }
 
-class SimpleTableIdGenerator : TableIdGenerator {
-    override fun toTableIdentifier(stream: DestinationStream.Descriptor): TableIdentifier =
-        tableIdOf(stream.namespace!!, stream.name)
+class SimpleTableIdGenerator(private val s3DataLakeConfiguration: S3DataLakeConfiguration) :
```

Comment:
nit: I'd rather pass just the namespace here, so it's less tied to the s3/glue stuff